tm2/pkg/bft/blockchain/pool_test.go

```diff
@@ -79,7 +79,7 @@ func TestBlockPoolBasic(t *testing.T) {
 	errorsCh := make(chan peerError, 1000)
 	requestsCh := make(chan BlockRequest, 1000)
 	pool := NewBlockPool(start, requestsCh, errorsCh)
-	pool.SetLogger(log.TestingLogger())
+	pool.SetLogger(log.NewNoopLogger())
```

Comment:
I think we should keep a TestingLogger here.

Reply:
I've replied to @gfanton on the same discussion here:

https://github.com/gnolang/gno/pull/1302#discussion_r1461608409

Reply:
I've talked to @gfanton and he convinced me this has huge benefits for our CI (testing logger that drops stuff into a file, and is served as an artifact)

I'll add in the support and ping in this thread

Reply:
Added the testing logger here:
[01e4fd5](https://github.com/gnolang/gno/pull/1302/commits/01e4fd51650544b9568ca5a016e8c1c43523f27a)